generator/src/main/java/com/scottlogic/deg/generator/fieldspecs/FieldSpecFactory.java

```diff
@@ -19,6 +19,7 @@ public FieldSpec construct(AtomicConstraint constraint) {
     }
 
     public FieldSpec toMustContainRestrictionFieldSpec(Collection<AtomicConstraint> constraints) {
+        if (constraints.isEmpty()) { return FieldSpec.Empty; }
```

Comment:
```suggestion
        if (constraints.isEmpty()) { 
          return FieldSpec.Empty; 
        }
```

Can we strike for this consistent format please.